public/components/notebooks/components/paragraph_components/paragraphs.tsx

```diff
@@ -252,25 +263,25 @@ export const Paragraphs = forwardRef((props: ParagraphProps, ref) => {
     props.setPara(newPara);
   };
 
-  // do not show output if it is a visualization paragraph and visInput is not loaded yet
-  const paraOutput = (!para.isVizualisation || visInput) && (
-    <ParaOutput
-      http={http}
-      pplService={pplService}
-      key={para.uniqueId}
-      para={para}
-      visInput={visInput}
-      setVisInput={setVisInput}
-      DashboardContainerByValueRenderer={DashboardContainerByValueRenderer}
-    />
-  );
+  const paraOutput =
```

Comment:
The pre-condition was causing display-logic problems.  pre-conditions moved into locations of `paraOutput` usage.